core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Util.java

```diff
@@ -101,6 +107,17 @@ public static Map<String, String> authHeaders(String token) {
     }
   }
 
+  public static Map<String, String> basicAuthHeader(String credential) {
```

Comment:
I think this still needs to be plural.

Reply:
I've adjusted this to be plural to be in line with `authHeaders` but it feels weird that those are plural, since technically there's always only a single authorization header being added to the map